### ISSUE TITLE
v1.0.1 creates git log warnings that fail mkdocs strict mode

### ISSUE THREAD

**Issue by andrewthauer** [timvink/mkdocs-git-revision-date-localized-plugin#89]
After upgrading to `v1.0.1` we started seeing build failures with the following:

```
WARNING  -  [git-revision-date-localized-plugin] Unable to read git logs of '/input/docs/.../my-doc.md'. Is git log readable? Option 'fallback_to_build_date' set to 'true': Falling back to build date
```

Is it possible #88 is causing these issues? It's a bit odd since we do fetch the entire history, but the warning says it's falling back to the build date for some reason. In any case, since it's a warning, if you are using `mkdocs build --strict` it will fail.

**Comment by timvink** [comment]
Thanks for reporting, will look into it in 1-2 weeks or so (currently on holiday)

**Comment by timvink** [comment]
Did you verify yet?

You mentioned "build failures" but you are showing a warning (it shouldn't break the build).

**Comment by andrewthauer** [comment]
@timvink - Sorry, I wasn't able to track down the root cause yet, but I'm pretty sure it's not related to the `v1.0.1 change. I'll close this and re-open if needed.

> You mentioned "build failures" but you are showing a warning (it shouldn't break the build).

This fails a build only used with the mkdocs `--strict` flag, since all warnings are treated as failures. We've temporarily disabled strict mode to workaround this.

**Comment by ssbarnea** [comment]
@andrewthauer @timvink Why is this ticket closed as the current bug prevents people from using strict mode in their pipelines, something that is known to be a good practice.

**Comment by andrewthauer** [comment]
@ssbarnea I wasn't able to confirm the issue was related to this plugin. Can you reproduce and confirm it is?

**Comment by ssbarnea** [comment]
Yes. I disabled the plugin for now.

**Comment by andrewthauer** [comment]
It could also be an issue with how `mkdocs` treats warnings in newer versions. If I recall there didn't seem to be any code changes to this plugin that seemed like they would have caused this. Disabling the plugin might be a red herring. But again, not entirely sure.

**Comment by ssbarnea** [comment]
I did not use mkdocs for very long time, so I cannot say if they changed something recently. Still, I almost always tried to run tools with strict mode (like sphinx), so I would not miss problems. Still, gue to the nature of this plugin, if you add `strict: true` you will not be able to test locally as it will cause failures for any uncommitted change that you have.

How this can be sorted? I am not sure. Maybe there is an info/notice level that does not become an error when you run in strict?

For the moment I disabled the plugin because it did not add essential functionality for me. For me it was more important to be able to have a default strict mode.
